client/components/repository/common/Comments/common.js

```diff
@@ -0,0 +1,39 @@
+import { mapActions, mapGetters, mapMutations } from 'vuex';
```

Comment:
In general, there is no need for this mixin since functionality can be only used for ActivityDiscussion component 😉